app/lib/shared/logging.dart

```diff
@@ -31,7 +31,7 @@ void setupDebugEnvBasedLogging() {
   }
   _envBasedLoggingSetupDone = true;
   // ignore: invalid_use_of_visible_for_testing_member
-  final debugEnv = envConfig.debug ?? '*';
+  final debugEnv = envConfig.debug ?? 'fake_server pub.email';
```

Comment:
Please leave a comment explaining that this is a hack for integration tests to work.

Or maybe, we should change the integration test fake email to use `print()` for the line it needs to output for the tests to work?